### PR TITLE
Adding Ipad model switcher

### PR DESCRIPTION
**Pull request by JoseMoreville**
This Pull request aims to change the Ipad model identifier by reading pc.ipadModel key value on playcover.plist.

### CODE REVIEW

**Comment by JoseMoreville** [comment]
Please, do not merge it until there's a new version of playtools

File: PlayTools/PlayLoader.m [hunk 1]
```diff
@@ -32,17 +32,21 @@
...
+#define OEM_ID ([[[PlaySettings shared] GET_OEM_ID] UTF8String])
 
 static int my_uname(struct utsname *uts) {
   int result = 0;
-  NSString *nickname = @"ipad";
-//   NSString *productType = @"iPad13,8";
-  NSString *productType = @"iPad8,6";
+    NSString *nickname = @"ipad";
+  //   NSString *productType = @"iPad13,8";
+    NSString *productType =  [[PlaySettings shared] GET_IPAD_MODEL];
```
Comment (KhoraLee) [inline PlayTools/PlayLoader.m:49]: Does this `NSString` really needed?
Looks strncpy with `DEVICE_MODEL` can do same thing (at line 54)

Comment (JoseMoreville) [reply PlayTools/PlayLoader.m:49]: > Does this `NSString *productType` really needed? Looks strncpy with `DEVICE_MODEL` can do same thing (at line 54)
@KhoraLee 
It was like that before, i know can just use DEVICE_MODEL, but otherwise it crashes for some reason.

Comment (KhoraLee) [reply PlayTools/PlayLoader.m:49]: I was asking because `strncpy(uts->machine, DEVICE_MODEL, strlen(DEVICE_MODEL) + 1);` runs without any issue to me.
image

Comment (JoseMoreville) [reply PlayTools/PlayLoader.m:49]: Mhh Let me try again

Comment (JoseMoreville) [reply PlayTools/PlayLoader.m]: Ok, It worked now
